src/tests/Microsoft.PowerFx.Interpreter.Tests.Shared/JsonSerializeUOTests.cs

```diff
@@ -0,0 +1,173 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Core.Tests;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+#pragma warning disable CA1065
```

Comment:
Do we need to re-enable it at the end of the file, or is the scope of the #pragma directive the file itself?

Reply:
Seems to be per file
[https://stackoverflow.com/questions/40044067/what-is-the-scope-of-a-c-sharp-pragma-directive](https://stackoverflow.com/questions/40044067/what-is-the-scope-of-a-c-sharp-pragma-directive)